Sources/StreamChatUI/Utils/DateFormatter+Extensions.swift

```diff
@@ -21,4 +21,20 @@ extension DateComponentsFormatter {
         df.unitsStyle = .full
         return df
     }()
+    
+    static let videoDuration: DateComponentsFormatter = {
```

Comment:
```suggestion
    static let videoDurationFormatter: DateComponentsFormatter = {
```

Reply:
when the static instance is declared within the type the suffix can be omitted without loosing the readability (like `NotificationCenter.default` or `UIApplication.shared`). There's also a `DateComponentsFormatter.minutes` right above so I'd not include the suffix tbh